src/seedu/addressbook/commands/HelpCommand.java

```diff
@@ -15,14 +15,15 @@
     public CommandResult execute() {
         return new CommandResult(
                 AddCommand.MESSAGE_USAGE
-                + "\n" + DeleteCommand.MESSAGE_USAGE
-                + "\n" + ClearCommand.MESSAGE_USAGE
-                + "\n" + FindCommand.MESSAGE_USAGE
-                + "\n" + ListCommand.MESSAGE_USAGE
-                + "\n" + ViewCommand.MESSAGE_USAGE
-                + "\n" + ViewAllCommand.MESSAGE_USAGE
-                + "\n" + HelpCommand.MESSAGE_USAGE
-                + "\n" + ExitCommand.MESSAGE_USAGE
+                        + "\n" + DeleteCommand.MESSAGE_USAGE
+                        + "\n" + ClearCommand.MESSAGE_USAGE
+                        + "\n" + FindCommand.MESSAGE_USAGE
+                        + "\n" + ListCommand.MESSAGE_USAGE
+                        + "\n" + SortCommand.MESSAGE_USAGE
+                        + "\n" + ViewCommand.MESSAGE_USAGE
+                        + "\n" + ViewAllCommand.MESSAGE_USAGE
+                        + "\n" + HelpCommand.MESSAGE_USAGE
+                        + "\n" + ExitCommand.MESSAGE_USAGE
```

Comment:
I believe the original indentation is fine? 
Be careful of unnecessary or accidental changes here as it can cause unnecessary merge conflict.